### PR TITLE
Improve documentation for detectors module

### PR DESCRIPTION
**Pull request by matthew-brett**
How about this?

### CODE REVIEW

**Comment by matthew-brett** [comment]
@stefanv - could you have a look?

File: code/detectors.py [hunk 1]
```diff
@@ -3,6 +3,11 @@
 These functions take a vector of values, and return a boolean vector of the
 same length as the input, where True indicates the corresponding value is an
 outlier.
+
+The outlier detection routines will likeley be adapted to the specific measure
```
Comment (stefanv) [inline code/detectors.py]: Spelling is likeley wrong

**Review by stefanv — CHANGES_REQUESTED** [review]
This looks good, just one minor nitpick.

**Comment by matthew-brett** [comment]
Fixed - is it OK now?

**Comment by stefanv** [comment]
Yeap, thanks a lot, Matthew!

**Pull request merged**